### PR TITLE
docs: Fixing a typo in 03-installation.mdx and improving grammar in 2 other files for readability

### PR DESCRIPTION
**Pull request by nelsonuprety1**
## What changed / motivation ?

In this pull request, I corrected a typo identified in the 03-installation.mdx file. Additionally, I addressed grammar issues in two other files to improve the overall readability of the document.

## Additional Context

- The typo fix in the 03-installation.mdx file relates to a critical step in the installation process, improving user guidance.
- Grammar corrections in the two additional files will make the content more readable.
- These changes align with the project's style guide and documentation standards, ensuring consistency across the codebase.

## Pre-flight checklist

- [X] I have read the contributing guidelines
      [Contribution Guidelines](../CONTRIBUTING.md)
- [X] Performed a self-review of my code

### CODE REVIEW

**Comment by nelsonuprety1** [comment]
I have successfully signed the Contributor License Agreement. Thank You.

File: apps/docs/docs/learn/02-thinking-in-stylex.mdx [hunk 1]
```diff
@@ -46,11 +46,11 @@ Most existing solutions to this problem rely on rules and conventions.
 <details>
   <summary>Utility Classes</summary>
   Atomic utility class names like Tailwind CSS and Tachyons rely on conventions and lint rules
-  to ensure that conflicting class names are not applied on the same element. Such tooling adds
+  to ensure that conflicting class names are not applied to the same element. Such tooling adds
```
Comment (nmn) [inline apps/docs/docs/learn/02-thinking-in-stylex.mdx:49]: It was originally correct.

File: apps/docs/docs/learn/02-thinking-in-stylex.mdx [hunk 2]
```diff
@@ -162,7 +162,7 @@ function MyComponent({style}) {
 This is a little more code, but the runtime cost is still minimal
 because of how fast the `stylex.props()` function is.
 
-Most other styling solutions don't enable composition of styles across file
+Most other styling solutions don't enable the composition of styles across file
```
Comment (nmn) [inline apps/docs/docs/learn/02-thinking-in-stylex.mdx:165]: This `the` changes the meaning and is wrong here.

**Review by nmn — CHANGES_REQUESTED** [review]
Some of the changes are regressions. I've commented on them

**Comment by nelsonuprety1** [comment]
@nmn i have updated the files with the changes that you suggested. Please kindly check thank you.

**Review by nmn — APPROVED** [review]

**Pull request merged**